src/rollup/utils/date-valid.js

```diff
@@ -0,0 +1,26 @@
+/** @module pathfora/utils/date-valid */
+
+/**
+ * Validate that the string is a properly formatted email
+ *
+ * @exports dateValid
+ * @params {string} date
+ * @params {string} max
+ * @params {string} min
+ * @returns {boolean} valid
+ */
+export default function dateValid (date, max, min) {
```

Comment:
```suggestion
export default function isValidDate (date, max, min) {
```

Reply:
Going to keep this name as is for consistency with `emailValid`.